easybuild/easyconfigs/p/PyTorch/PyTorch-1.4.0-fosscuda-2019b-Python-3.7.4.eb

```diff
@@ -268,32 +268,23 @@ dependencies = [
     ('NCCL', '2.4.8'),
 ]
 
-download_dep_fail = True
-
-prebuildopts = 'PYTORCH_BUILD_VERSION=%(version)s PYTORCH_BUILD_NUMBER=1 VERBOSE=1 '
-prebuildopts += 'MAX_JOBS=%(parallel)s '
-prebuildopts += 'LDFLAGS="$LDFLAGS -ldl" '
-prebuildopts += 'USE_FFMPEG=ON USE_IBVERBS=1 USE_GFLAGS=ON USE_GLOG=ON '
-prebuildopts += 'CUDNN_LIB_DIR=$EBROOTCUDNN/lib64 CUDNN_INCLUDE_DIR=$EBROOTCUDNN/include '
-prebuildopts += 'USE_SYSTEM_NCCL=1 NCCL_INCLUDE_DIR=$EBROOTNCCL/include '
-prebuildopts += 'TORCH_CUDA_ARCH_LIST="3.5 3.7 5.2 6.0 6.1 7.0 7.2 7.5" '
```

Comment:
Shouldn't this be replaced with:

```python
# default CUDA compute capabilities to use (override via --cuda-compute-capabilities)
cuda_compute_capabilities = ['3.5', '3.7', '5.2', '6.0', '6.1', '7.0', '7.2', '7.5']
```

to make it equivalent with what we had before?

Reply:
done